bigbluebutton-html5/imports/ui/components/user-list/user-list-content/user-participants/user-list-item/user-dropdown/component.jsx

```diff
@@ -265,7 +265,7 @@ class UserDropdown extends Component {
       ));
     }
 
-    if (allowedToUnmuteAudio) {
+    if (allowedToUnmuteAudio && user.id === currentUser.id) {
```

Comment:
I think we should also remove https://github.com/bigbluebutton/bigbluebutton/blob/282fd4055dbc0c55aa96d87edcefb4d6670635d6/bigbluebutton-html5/imports/ui/components/user-list/service.js#L16

Reply:
I think removing ALLOW_MODERATOR_TO_UNMUTE_AUDIO from the user list would fix the problem at the source and would mean that the changes to the user-dropdown would be unnecessary because "allowedToUnmuteAudio" would end up being false in user-list.